### DISCUSSION TITLE
Time for a support ticket to get a response

### DISCUSSION THREAD

**Discussion by Daeraxa** [community/community#40729] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Just wondering what kind of lead time is there to have a ticket responded to by support?
It has been open for over two months now without any comment at all.

**Comment by radar-blip** [comment]
If its been that long (without response) you may want to submit another ticket. 
Good luck!

**Reply by risico** [reply]
I've been waiting for 18 days and counting.
